subsys/bluetooth/host/iso.c

```diff
@@ -723,24 +723,6 @@ static uint16_t iso_chan_max_data_len(const struct bt_iso_chan *chan,
 	/* Update the max_data_len to take the max_controller_data_len into account */
 	max_data_len = MIN(max_data_len, max_controller_data_len);
 
-	/* Since this returns the maximum ISO data length size, we have to take
-	 * the header size into account, as that also needs to be inserted into
-	 * the SDU
-	 */
-	if (ts == BT_ISO_TIMESTAMP_NONE) {
```

Comment:
I seem to remember this was added to make sure a buffer was not overrun in an border-case EBQ test. Should the check be removed for both cases here, or just the case where ts != BT_ISO_TIMESTAMP_NONE?

Reply:
The timestamp is not related to the SDU size (timestamp is part of the header, and not the SDU), so should just never have been added. 

If there's an issue with an EBQ tests without this, then the issue lies elsewhere